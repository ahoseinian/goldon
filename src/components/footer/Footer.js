import React from 'react'
import styled from 'styled-components'
import {color} from '../theme'
import Social from '../Contact/Social'
import Delivery from './Delivery'

const Wrapper = styled.div `
  margin-top: 2rem;
`
const CopyRight = styled.div `
  color: ${color.black};
  margin-left: 1rem;
  font-size: 1.5rem;
`
const Footer = () => (
  <Wrapper className="container">
    <div className="row">
      <div className="col-md-8">
        <Delivery />
      </div>
      <div className="col-md-4 d-flex justify-content-between align-items-center mb-3">
        <Social/>
        <CopyRight> © میاریم</CopyRight>
      </div>
    </div>
  </Wrapper>
)

export default Footer;
