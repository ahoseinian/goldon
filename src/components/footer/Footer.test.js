import React from 'react'
import {shallow} from 'enzyme'
import Footer from './Footer'
import toJson from 'enzyme-to-json';

describe('Footer', () => {
  it('renders', () => {
    const wrapper = shallow(<Footer />)
    expect(
      toJson(wrapper)
    ).toMatchSnapshot()
  })
})
