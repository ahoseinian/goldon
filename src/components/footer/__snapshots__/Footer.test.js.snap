exports[`Footer renders 1`] = `
<styled.div
  className="container">
  <div
    className="row">
    <div
      className="col-md-8">
      <Delivery />
    </div>
    <div
      className="col-md-4 d-flex justify-content-between align-items-center mb-3">
      <Social />
      <styled.div>
         © میاریم
      </styled.div>
    </div>
  </div>
</styled.div>
`;
